app/src/main/java/live/ditto/dittotoolsapp/MainActivity.kt

```diff
@@ -37,7 +38,7 @@ class MainActivity : ComponentActivity() {
         lateinit var ditto: Ditto
         try {
             val androidDependencies = DefaultAndroidDittoDependencies(applicationContext)
-            val identity = DittoIdentity.OnlinePlayground(androidDependencies, appId = "YOUR_APPID", token = "YOUR_TOKEN", enableDittoCloudSync = true)
+            val identity = DittoIdentity.OnlinePlayground(androidDependencies, appId = "c5adf242-f3fc-4987-aac9-3cb79967274e", token = "812872a2-28a5-4ee7-b487-04f452eb6176", enableDittoCloudSync = true)
```

Comment:
Is the AppId/token addition intentional?

Reply:
Hi, 
Good catch,
It is not intentional, I change those value to test the app. 
It should not be part of the PR. 
I will remove it.

Thanks